src/Sulu/Bundle/DocumentManagerBundle/Resources/config/behaviors.xml

```diff
@@ -4,6 +4,11 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
+    <parameters>
+        <parameter key="sulu_document_manager.subscriber.alias.class">Sulu\Component\DocumentManager\Subscriber\Behavior\Path\AliasFilingSubscriber</parameter>
+        <parameter key="sulu_document_manager.subscriber.filing.class">Sulu\Component\Content\Document\Subscriber\StructureTypeFilingSubscriber</parameter>
+        <parameter key="sulu_document_manager.subscriber.behavior.reset.path.class">Sulu\Component\DocumentManager\Subscriber\Behavior\Path\ResetFilingPathSubscriber</parameter>
```

Comment:
We haven't used parameters in this file yet, and since symfony also don't use parameters this way starting with 3.0, I think we  should not start that now.